genesis/process/genesisBlockCreator_test.go

```diff
@@ -415,6 +425,52 @@ func TestNewGenesisBlockCreator(t *testing.T) {
 		require.True(t, errors.Is(err, genesis.ErrNilSmartContractParser))
 		require.Nil(t, gbc)
 	})
+	t.Run("nil RunTypeComponents should error", func(t *testing.T) {
+		t.Parallel()
+
+		arg := createMockArgument(t, "testdata/genesisTest1.json", &mock.InitialNodesHandlerStub{}, big.NewInt(22000))
+		arg.RunTypeComponents = nil
+
+		gbc, err := NewGenesisBlockCreator(arg)
+		require.True(t, errors.Is(err, mxErrors.ErrNilRunTypeComponents))
```

Comment:
optional: there is also `require.ErrorIs`

Reply:
fixed